app/routes.py

```diff
@@ -54,7 +54,13 @@ def logout():
 
 
 # REST resource APIs
+@app.route('/users/<user_id>/movie-preferences', methods=['GET'])
```

Comment:
I think REST convention is to have a `/` trailing endpoints that return lists

Reply:
(Unlike the post, where it's creating a single thing)

Reply:
Ooooh, good catch